db/python/tables/sample.py

```diff
@@ -402,6 +417,14 @@ async def get_sample_with_missing_participants_by_internal_id(
         )
         return {row['id']: row['external_id'] for row in rows}
 
+    async def get_samples_create_date(self, sample_ids: list[int]):
```

Comment:
Nitpick type annotation:

Might need to import datetime for this annotation to work though
```suggestion
    async def get_samples_create_date(self, sample_ids: list[int]) -> dict[int, date]:
```